### DISCUSSION TITLE
Typo in explanation of Actions approval requirement

### DISCUSSION THREAD

**Discussion by ajsirota** [community/community#4132] [Mobile] [answered]
On the PR page, the explanation that PRs by first-timers need maintainer approval has a few grammatical errors. 
![image](https://user-images.githubusercontent.com/67526318/121112883-f5210a00-c7c5-11eb-8b9f-2661b6087664.jpeg)
I would suggest replacing it with:
> First-time contributors need maintainer approval to run Actions workflows. Learn more [link]

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @ajs256! We'll fix it ASAP!

**Reply by aconsuegra** [reply]
And the fix is in. It will be included in our upcoming update!
